Source/debug.cpp

```diff
@@ -242,44 +242,19 @@ std::string DebugCmdLevelUp(const std::string_view parameter)
 	return "New experience leads to new insights.";
 }
 
-std::string DebugCmdGetAllSpells(const std::string_view parameter)
-{
-	SetSpellLevelCheat(SPL_FIREBOLT, 8);

```

Comment:
`SetSpellLevelCheat()` is unused now and should be fully removed.

Reply:
oh I can actually reuse it : D